### ISSUE TITLE
Broken Contact Us link on glider DAC page on ioos.us

### ISSUE THREAD

**Issue by robragsdale** [ioos/ioos-us#71]
The "contact us" link under "about" does not work in Chrome and does not show up in IE.  The link on the [glider DAC page](https://gliders.ioos.us/index.html) is broken, but is working from the home page [https://ioos.us/](url).

**Comment by bnfish** [comment]
@robragsdale are you referring to the link here: https://gliders.ioos.us/index.html?

if yes, follow up question: are you able to access this repository: https://github.com/ioos/gliders?

**Comment by robragsdale** [comment]
@bnfish Yes, I am referring to https://gliders.ioos.us/index.html and I can access the repository https://github.com/ioos/gliders.
